### ISSUE TITLE
Error converting avif file with a clap box

### ISSUE THREAD

**Issue by FrankGalligan** [strukturag/libheif#374]
my command:
./examples/heif-convert ./kimono.crop.avif ./out.png

Link to file:
https://github.com/AOMediaCodec/av1-avif/blob/master/testFiles/Link-U/kimono.crop.avif

Output:
File contains 1 images
Could not decode image: 0: Invalid input: Invalid clean-aperture specification

**Comment by farindk** [comment]
I relaxed the input check for the clap box. It now decodes the file successfully.
However, is it expected that output only has size 1x3 ?

**Comment by FrankGalligan** [comment]
No they want it to look like this:
https://resources.link-u.co.jp/avif/decoded/kimono.crop.png

Maybe the file is bad, but then I tried another file here:
https://github.com/AOMediaCodec/av1-avif/blob/master/testFiles/Microsoft/Chimera_8bit_cropped_480x256.avif

And that file had pretty simple clap. The values I parsed in libavif and they seemed reasonable.

But was still getting an error in heif-convert.

I looked into it more and I think the clap parse is bad.
I made a PR #375 which should fix it. (Looks like it is failing lint)
